src/test/java/edu/kis/powp/jobs2d/TestJobs2dPatterns.java

```diff
@@ -32,6 +41,36 @@ private static void setupPresetTests(Application application) {
 
 		application.addTest("Figure Joe 1", selectFirstTestFigureOptionListener);
 		application.addTest("Figure Joe 2", selectSecondTestFigureOptionListener);
+		application.addTest("draw rectangle",event -> {
```

Comment:
It is good to keep a convention used in the original code (class instance instead of lambda)